app/src/main/java/io/github/droidkaigi/confsched2018/di/NetworkModule.kt

```diff
@@ -37,7 +37,7 @@ internal object NetworkModule {
     fun provideRetrofit(okHttpClient: OkHttpClient): Retrofit {
         return Retrofit.Builder()
                 .client(okHttpClient)
-                .baseUrl("https://droidkaigi.jp/2018/sessionize/")
+                .baseUrl("https://droidkaigi.jp/2018/")
```

Comment:
To prove these changes, let me attach one screenshot.

<img width="300" src="https://user-images.githubusercontent.com/4340693/35052520-1ce49048-fbeb-11e7-8a43-ed34d1ab0347.jpg" >